modules/programs/firefox.nix

```diff
@@ -745,6 +757,15 @@ in {
 
     programs.firefox.finalPackage = wrapPackage cfg.package;
 
+    programs.firefox.policies = {
+      ExtensionSettings = listToAttrs (map (lang:
+        nameValuePair "langpack-${lang}@firefox.mozilla.org" {
+          installation_mode = "normal_installed";
+          install_url =
+            "https://releases.mozilla.org/pub/firefox/releases/${cfg.package.version}/linux-x86_64/xpi/${lang}.xpi";
```

Comment:
I just noticed, that `cfg.package` has a nullable type. Is that a problem?

Reply:
Unfortunately yes. Maybe an assertion should be added to inform the user that the `languagePacks` option does not work when `package` is set to null.